x-pack/plugins/task_manager/server/config.ts

```diff
@@ -6,6 +6,7 @@
 
 import { schema, TypeOf } from '@kbn/config-schema';
 
+export const MAX_MAX_WORKERS = 100;
```

Comment:
Total nit. but... can we maybe name it something like `MAX_WORKERS_LIMIT`?
`MAX_MAX_WORKERS` feels like a typo 😆

Reply:
thx for that - I like yours better, mine did seem a bit odd :-)